### ISSUE TITLE
DBC, maps, mmaps and vmaps

### ISSUE THREAD

**Issue by argumumble** [dufernst/FirestormWoD#8]
It seems like you cannot extract those required files from the WoDEmpire client, could someone please upload them somewhere? 

I cannot get hold of a client where I can extract them from either, since the Firestorm one doens't seem to be "extractable"

Thanks!

**Comment by argumumble** [comment]
I forgot to mention that I already have the files from some repack, but they must be the wrong version since the worldserver.exe crashes when it’s looking for them.

Therefor I need the correct ones to run this server. And I’d appriciate if I could have a link to the correct ones!

**Comment by dufernst** [comment]
You can download the full client from here:
http://forum.wowcircle.com/en/showthread.php?t=4088

Extracting from it should work just fine.

**Comment by argumumble** [comment]
I'll try that client, thank you!

And if I remeber correct, I just put the extractors and ,dll:s in the root-folder of the client? Or do I have to put them elsewhere?

**Comment by dufernst** [comment]
Indeed in the root folder.

**Comment by portunus** [comment]
Check the guide from trinitycore it should be roughly the same. https://trinitycore.atlassian.net/wiki/spaces/tc/pages/10977329/Windows+Core+Installation

**Comment by argumumble** [comment]
My extractors closes right away when I run them, I tried both WoWCircle, Firestorm and WoDEmpire...

Could someone please provide these things? :(

**Comment by portunus** [comment]
You can produce the files and compile the core with stock trinitycore? Only thing i can think of is that you could perhaps run it as administrator or look at troubleshooting @ trinitycore

**Comment by argumumble** [comment]
I managed to extract the files, but now the worldserver.exe crashes right away when reading the DataDir part. And I've set the DataDir to "." and tried making a folder named "Data" and placed them in there and changed in the config so it will be looking into that folder, but it still crashes.

Sorry for so many questions, but I am a complete noob here :D 

I doesn't provide me any logs at all so I cannot do much really, any ideas?
